packages/cli/tests/build.test.js

```diff
@@ -182,8 +182,10 @@ describe('preact build', () => {
 			`${dir}/build/${bundleFile}`,
 			'utf8'
 		);
+
 		// "Hello World!" should replace 'process.env.PREACT_APP_MY_VARIABLE'
-		expect(transpiledChunk.includes('console.log("Hello World!")')).toBe(true);
+		expect(transpiledChunk.includes('Hello World!')).toBe(true);
```

Comment:
nit: We do need to check for quotes when replacing env vars, else if we fail to stringify we'd end up with `var foo = Hello World!`

```suggestion
		expect(transpiledChunk.includes('"Hello World!"')).toBe(true);
```